src/django/api/fields.py

```diff
@@ -9,6 +9,33 @@
 from rest_framework.serializers import ValidationError
 
 
+def get_polygon_geometry(geojson):
+    if type(geojson) in [str, bytes]:
+        geojson = json.loads(geojson)
```

Comment:
Do you want to `try` and raise a ValidationError? Uploading bad data generates an unhandled `UnicodeDecodeError`.

Reply:
Nice catch. Wrapped in `try` in [ac3b493](https://github.com/azavea/iow-boundary-tool/pull/223/commits/ac3b4935671c5b69faea34c0430280e2493f461e)